docs/en/engines/table-engines/mergetree-family/invertedindexes.md

```diff
@@ -35,26 +35,26 @@ useful the resulting full-text index will be.
 </div>
 
 :::note
-Full-text indexes are experimental and should not be used in production environments yet. They may change in the future in backward-incompatible
+Text indexes are experimental and should not be used in production environments yet. They may change in the future in backward-incompatible
 ways, for example with respect to their DDL/DQL syntax or performance/compression characteristics.
 :::
 
 ## Usage {#usage}
 
-To use full-text indexes, first enable them in the configuration:
+To use text indexes, first enable them in the configuration:
 
 ```sql
 SET allow_experimental_full_text_index = true;
 ```
 
-An full-text index can be defined on a string column using the following syntax
+An text index can be defined on a string column using the following syntax
 
 ```sql
 CREATE TABLE tab
 (
     `key` UInt64,
     `str` String,
-    INDEX inv_idx(str) TYPE text(tokenizer = 'default|ngram|split|no_op' [, ngram_size = N] [, separators = []] [, max_rows_per_postings_list = M]) GRANULARITY 1
+    INDEX inv_idx(str) TYPE text(tokenizer = 'default|ngram|split|no_op' [, ngram_size = N] [, separators = []] [, max_rows_per_postings_list = M]) [GRANULARITY 64]
```

Comment:
Minor: I think instead of `64` we want a placeholder here, e.g. `GRANULARITY gran`. L. 80 can then reference the placeholder.